test/util/date-time.js

```diff
@@ -40,12 +40,16 @@ afterEach(() => {
 // Returns an ISO string for a date in the past that is no earlier than the
 // specified dates.
 export const fakePastDate = (dateStrings) => {
-  const dateTimes = dateStrings
+  const parsed = dateStrings
     .filter(s => s != null)
-    .map(s => DateTime.fromISO(s));
-  if (dateTimes.length === 0) return faker.date.past().toISOString();
-  const from = DateTime.max(...dateTimes);
-  const now = DateTime.local();
-  if (from > now) throw new Error('future date');
-  return faker.date.between({ from: from.toISO(), to: now.toISO() }).toISOString();
+    .map(s => Date.parse(s));
+  if (parsed.length === 0) return faker.date.past().toISOString();
+  const from = Math.max(...parsed);
+  const to = Date.now() - 1000;
```

Comment:
`- 1000` is a change from the current behavior, since `now` was used before, and that went up to the current millisecond. I noticed that `faker.date.past()` does `- 1000`, and I thought it made sense to try to match it: https://github.com/faker-js/faker/blob/549d71cf3361cca787d685a1c053ef8135e13641/src/modules/date/index.ts#L88